drivers/staging/android/lowmemorykiller_tng.c

```diff
@@ -161,8 +161,8 @@ static void calc_params(struct calculated_params *cp, gfp_t mask)
 	int i;
 	int array_size;
 	long free_swap = get_nr_swap_pages();
-	long irp = global_node_page_state(NR_INDIRECTLY_RECLAIMABLE_BYTES)
-	  >> PAGE_SHIFT;
+	long irp = global_node_page_state(NR_SLAB_RECLAIMABLE) +
+	    global_node_page_state(NR_KERNEL_MISC_RECLAIMABLE);
```

Comment:
I'll rephrase my previous comment: why are you also adding NR_SLAB_RECLAIMABLE here? (you omit it in `print_obituary(...)`)

Reply:
I replicated the changes done in b29940c1abd7a4c3abeb926df0a5ec84d6902d47.
In the commit NR_INDIRECTLY_RECLAIMABLE_BYTES was replaced with NR_SLAB_RECLAIMABLE +NR_KERNEL_MISC_RECLAIMABLE.

Reply:
It's also replaced without slab in the same commit here: https://github.com/sonyxperiadev/kernel/commit/b29940c1abd7a4c3abeb926df0a5ec84d6902d47?diff=unified#diff-fa19cdd9c3369d7f59aa2e8404628109408dbf8e1b568d1157a27328f75b8410R681

Reply:
Makes sense do it in the same way, adjusted.

Reply:
I've tested both approaches, the latter makes lmk to be less aggressive. Approving _post-mortem_:)

Reply:
> For future ref: variable `ipr` means Indirectly Reclaimable Pages, which should've been renamed in the light of ported changes.


I was about to do that but the PR was already merged.

Reply:
@Thaodan open a new PR